src/sections/GridSection.stories.tsx

```diff
@@ -74,6 +74,8 @@ export const FourColumnsEdit = () => (
     title="Very Long Section Title That Wraps on Mobile"
```

Comment:
It's interesting that we were missing the ability to have an edit link, and yet had an "edit" story - somewhere along the line this must have been a regression, but Detroit still has the edit piece locally.